build/yamls/antrea-migrator.yml

```diff
@@ -0,0 +1,55 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  labels:
+    app: antrea
+    component: antrea-migrator
+  name: antrea-migrator
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-migrator
+  template:
+    metadata:
+      labels:
+        app: antrea
+        component: antrea-migrator
+    spec:
+      hostPID: true
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/os: linux
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+      serviceAccountName: antrea-agent
+      volumes:
+        - name: containerd
+          hostPath:
+            path: /run/containerd
+      initContainers:
+        # initContainer will kill all sandboxes container
+        - name: antrea-migrator-init
+          image: antrea/antrea-ubuntu:latest
```

Comment:
Done, since `build/yamls/antrea-migrator.yml` is not generated automatically, I added one line in `hack/release/prepare-assets.sh` using `sed` command to change the image name.